astronomer.tf

```diff
@@ -10,19 +10,19 @@ resource "null_resource" "helm_repo" {
   }
 
   provisioner "local-exec" {
-    when    = "destroy"
+    when = "destroy"
     command = "rm -rf './helm.astronomer.io'"
   }
 }
 
 # this is for development use
 resource "helm_release" "astronomer_local" {
   depends_on = ["null_resource.helm_repo"]
-  name      = "astronomer"
-  version   = var.astronomer_version
-  chart     = "./helm.astronomer.io"
+  name = "astronomer"
+  version = var.astronomer_version
+  chart = "./helm.astronomer.io"
```

Comment:
do you want to terraform fmt it?

Reply:
Strange.. it doesn't look like `terraform fmt` is formatting this properly.

Reply:
then leave it, I don't really mind - mostly just don't want it to go back and forth. so whatever fmt lands on, we can leave it like that

Reply:
Open bug in Terraform > 0.12: https://github.com/hashicorp/terraform/issues/21434

I lost my mind over that too :D